backend/LoadCell.py

```diff
@@ -0,0 +1,104 @@
+from enum import Enum
+import matplotlib.pyplot as plt
+import json
+import numpy as np
+
+class CalLoadCell():
+    class State(Enum):
+        NORMAL = 0
+        STORE = 1
+        WAIT = 2
+
+    def __init__(self, name="DefaultLoadCell") -> None:
+        self.name = name
+        self.load_cell = 0
+        self.tare = 0
+        self.slope = 1
+        self.vol_offset = 0
+        self.cal = False
+        self.cal_points = {}
+        self.mass = 0
+        self.volts = 0
+        self.txtval = 0
+        self.current_state = self.State.NORMAL
+
+    def reset(self):
+        self.current_state = self.State.NORMAL
+
+    def _collect_point(self, voltage, mass):
+        self.current_state: self.State.NORMAL
+        if not self.cal_points:
+            self.cal_points = {0: voltage}
+            self.vol_offset = voltage
+        else:
+            self.cal_points[mass] = voltage
+
+    def _discard_cal(self, num=1):
```

Comment:
For this method and many others you are declaring them but never using them, which is fine as I understand they are meant to be used outside this module. However having the name use an underscore at the start of these methods implies they are private, meaning they will only be used inside this module.

Reply:
I unprivated this, I just wasn't sure if this was going to be used or not, and was looking for confirmation if we needed this function. I just added as a just in case

Reply:
sounds good we can leave it for now and remove it later, usually Id say remove it now but for now lets leave it